systemtests/aci_test.go

```diff
@@ -59,3 +59,48 @@ func (s *systemtestSuite) TestACIMode(c *C) {
 	c.Assert(s.cli.EndpointGroupDelete("default", "epgB"), IsNil)
 	c.Assert(s.cli.NetworkDelete("default", "aciNet"), IsNil)
 }
+
```

Comment:
All ACI specific test cases would come under this file as of now. TestACIMode was already present and I think we should add ACI specific test cases here in the same file.